src/lib/LDWorkbenchConfiguration.d.ts

```diff
@@ -50,23 +50,45 @@ export interface LDWorkbenchConfiguration {
          */
         batchSize?: number;
       };
-      generator: {
-        /**
-         * Path (prefixed with "file://") or SPARQL Query
-         * that makes the generator using SPARQL construct.
-         */
-        query: string;
-        /**
-         * The SPARQL endpoint for the generator.
-         * If it starts with "file://", a local RDF file is queried.
-         * If ommmitted the endpoint of the Iterator is used.
-         */
-        endpoint?: string;
-        /**
-         * Overrule the generator's behaviour of fetching results for 10 bindings of $this per request.
-         */
-        batchSize?: number;
-      };
+      /**
+       * @minItems 1
+       */
+      generator: [
+        {
+          /**
+           * Path (prefixed with "file://") or SPARQL Query
+           * that makes the generator using SPARQL construct.
+           */
+          query: string;
+          /**
+           * The SPARQL endpoint for the generator.
+           * If it starts with "file://", a local RDF file is queried.
+           * If ommmitted the endpoint of the Iterator is used.
+           */
+          endpoint?: string;
+          /**
+           * Overrule the generator's behaviour of fetching results for 10 bindings of $this per request.
+           */
+          batchSize?: number;
+        },
```

Comment:
I would name this type s.t. you don't have to repeat it. What do you think?

Reply:
ah this is automatically generated from the JSON schema here: https://github.com/netwerk-digitaal-erfgoed/ld-workbench/blob/main/static/ld-workbench.schema.json